src/main/java/cz/xtf/radanalytics/util/configuration/RadanalyticsConfiguration.java

```diff
@@ -0,0 +1,45 @@
+package cz.xtf.radanalytics.util.configuration;
+
+import cz.xtf.XTFConfiguration;
+
+public class RadanalyticsConfiguration extends XTFConfiguration {
+
+
+	private static final String IMAGE_PY_SPARK = "rad.driver.python";
+	private static final String IMAGE_JAVA_SPARK = "rad.driver.java";
+	private static final String IMAGE_SCALA_SPARK = "rad.driver.scala";
+	private static final String IMAGE_OSHINKO_REST = "rad.oshinko.rest";
+	private static final String IMAGE_OSHINKO_CLI = "rad.oshinko.cli";
+	private static final String IMAGE_OSHINKO_WEBUI = "rad.oshinko.webui";
+	private static final String SHOULD_BUILD_ALL_DRIVER_APPLICATION = "rad.build.all.drivers";
+
+	public static final String OSHINKO_WEBUI_RESOURCES_URL="https://raw.githubusercontent.com/radanalyticsio/oshinko-webui/master/tools/ui-template.yaml";
+	public static final String MONGODB_EPHEMERAL_TEMPLATE_URL = "https://raw.githubusercontent.com/openshift/origin/master/examples/db-templates/mongodb-ephemeral-template.json";
```

Comment:
These resources should be specified as properties, no need to have them hardcoded here. Add prefix TEMPLATE_ to this kind of properties